.github/workflows/update-wordpress-js-dependencies-orchestrator.yml

```diff
@@ -0,0 +1,67 @@
+name: Update WordPress JS Dependencies
```

Comment:
Seems like the name is the same as it is for `update-wordpress-js-dependencies.yml`. Does it make sense to maybe name this one `trigger-update-wordpress-js-dependencies`?